### PR TITLE
OpenMP updates

### PR DESCRIPTION
**Pull request by SwRaw**

### CODE REVIEW

**Comment by SwRaw** [comment]
@ronlieb Could you please review these changes? I don't know JP's id, hence requesting you to run it by the team for confirmation.

**Review by ronlieb — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,6 +43,29 @@ as follows:
 
 ## Filenames and folder structure
 
+Please use snake case for file names. Our documentation follows pitchfork for
```
Comment (saadrahim) [inline CONTRIBUTING.md:46]: Please separate these updates to the CONTRIBUTING.md file into a separate PR. This is a separate update from OpenMP.

**Review by saadrahim — DISMISSED** [review]

**Comment by samjwu** [comment]
> @samjwu The links to ROCProfilerV1 User Manual are not rendering correctly. I can't spot the problem. Could you please take a look.

The syntax that you used was correct. The only issue was that the rocprofiler 5.6.0 docs did not have that page. I have added it to that branch and rebuilt those docs as well as the docs for this PR.

The link is now visible at https://advanced-micro-devices-demo--2404.com.readthedocs.build/en/2404/reference/openmp/openmp.html

File: CONTRIBUTING.md (hunk 1 above)
Comment (SwRaw) [reply CONTRIBUTING.md:46]: I have made no changes to the CONTRIBUTING.md file, somehow it showed conflict so I just resolved them.

Comment (samjwu) [reply CONTRIBUTING.md]: This was a merge conflict. Fixed in https://github.com/RadeonOpenCompute/ROCm/pull/2404/commits/f58a6b55506ed433407b0c8783c7d6c85217b207

The content is already in `docs/contribute/feedback.md`.

**Review by samjwu — APPROVED** [review]
No changes in content, just formatting after Ron's approval

**Pull request merged**